### ISSUE TITLE
Latest release doesn't seem to work with Kvaser PCIE cards 

### ISSUE THREAD

**Issue by icolwell-as** [astuff/kvaser-deb#8]
Something seems to have broken with the latest updates to `linuxcan`, PCIE cards no longer work.
Maybe some changes are needed in how it is packaged up.

After installing, PCIE drivers seem non-existant:

```
$ sudo modprobe pcican
modprobe: FATAL: Module pcican not found in directory /lib/modules/5.4.0-74-generic
$ sudo modprobe pciefd
modprobe: FATAL: Module pciefd not found in directory /lib/modules/5.4.0-74-generic

```

**Comment by JWhitleyWork** [comment]
@icolwell-as If you wouldn't mind releasing the packages for Groovy, I'd be happy to help troubleshoot.

**Comment by icolwell-as** [comment]
Thanks for the offer @JWhitleyWork! , do you mean Ubuntu Groovy? Ubuntu 20.10? Groovy is EOL in July, so we won't release for that Ubuntu version. We are planning to move to socketCAN asap, so it might be a lost cause to debug this anyway.

**Comment by icolwell-as** [comment]
resolved by #10
